docs/_docs/cross-chain/bridge.md

```diff
@@ -0,0 +1,62 @@
+The value received by the bridge contract during the execution of `sendNative` is what will be logged as transfer to the other layer.
+
+## Layer 1 To Layer 2 Specifics
+
+The `ObscuroBridge.sol` contract is responsible for managing the layer 1 side of the bridge. It's the "bridge to Obscuro".
+The `EthereumBridge.sol` contract is responsible for managing the layer 2 side of the bridge. It's the "bridge to Ethereum".
+
+In order to bridge tokens over they need to be whitelisted. **Only accounts with the admin role can whitelist tokens!**
+
+When an asset is whitelisted, the bridge internally uses the `publishMessage` call on the MessageBus which can be found within the `ManagementContract`. In the message that is published the bridge "tells" the other side of it, which resides on layer 2 that a token has been whitelisted. This in turn creates a wrapped version of the token on the other side. This version of the token can only be minted and burned by the layer 2 bridge.
+
+ * Minting allows to create fresh funds on the L2 when they get locked on L1.
+ * Burning allows to destroy supply on L2 in order to release it from the bridge on L1.
+
+The protocol to bridge assets using `mint`/`burn` is based on the `MessageBus`'s `publishMessage` too. The `ObscuroBridge` tells the layer 2 that it has locked a certain amount of tokens and because of that address Y should receive the same amount of the wrapped token counterpart. This is when `mint` comes into play. On the other hand, the `EthereumBridge` burns tokens when received and tells the L1 that it has in fact done so. This is interpreted by the L1 smart contract as evidence to release locked funds.
+
+
+## Security
+
+The bridge inherits the `CrossChainEnabled` contract. It provides a modifier that can be attached to public smart contract functions in order to limit them. Normally you can limit a function to be called by `msg.sender`. With this new modifier you can limit a function to be called from the equivalent of `msg.crosschainsender`. 
+
+The bridge links its two sides together - the ObscuroBridge and the EthereumBridge and limits the cross chain sender to only be the opposite contract on the `receiveAssets` functions. This means that only the `CrossChainMessenger` can call them when it is provided with a valid cross chain message produced by the bridge. As the cross chain sender is determined by `msg.sender` at the point of calling `publishMessage`, no one else but the bridge has the capability of queueing a message that will pass.
+
+
+The layer 2 bridge works much the same way as the L1 bridge and uses the `MessageBus` to facilitate communication. 
+
+## Building Bridges
+
+The standard bridge was designed to work without requiring access to any private platform capabilities. This means that anyone can build a bridge using the same cross chain messaging API. There is no permissioning required to do so.
```

Comment:
mention that we encourage developers to build novel bridges based on liquidity pools or novel concepts.
And that we provide all the apis they need